test/inductor/test_aot_inductor.py

```diff
@@ -972,6 +993,7 @@ class AOTInductorTestABICompatibleCuda(TestCase):
     "abi_compatible_cuda",
     # test_failures, xfail by default, set is_skip=True to skip
     {
+        "test_arange_with_start": TestFailure(("abi_compatible_cuda",)),
```

Comment:
Wondering why this test would fail in the abi_compatible_cuda mode?

Reply:
because it requires `aoti_torch_normal_functional`